test/FunctionalTests/Microsoft.OData.Edm.Tests/EdmUtilTests.cs

```diff
@@ -13,6 +14,51 @@ namespace Microsoft.OData.Edm.Tests
     ///</summary>
     public class EdmUtilTests
     {
+        private static EdmEntityType customer = new EdmEntityType("NS", "Customer");
+        private static EdmEntityType vipCustomer = new EdmEntityType("NS", "VipCustomer", customer);

```

Comment:
>vipCustomer [](http://example.com/codeflow?start=37&length=11)

Do you assume the static member initialization order?

In Windows system, it could be static order, but, from C#
the static field initializers are executed at an implementation-dependent time prior to the first use of a static field of that class.
So, I think it would be better to do the initialization inside a static type initializer (static constructor).